packages/node-bridge/tsconfig.json

```diff
@@ -11,6 +11,6 @@
     "module": "commonjs",
     "skipLibCheck": true
   },
-  "include": ["helpers.ts", "bridge.js", "launcher.js"],
+  "include": ["helpers.ts", "bridge.js", "launcher.js", "web-handler.js"],
```

Comment:
Can you check the size before and after of thse files?

We need to make sure they are extremely small because any increase could put new deployments over the limit of 50MB.

Reply:
Good call.
Ncc output (I trimmed out some of the lines):

Main:
```
 1kB  helpers/helpers.d.ts
 2kB  helpers/types.d.ts
22kB  helpers/index.js
122kB  source-map-support/index.js
```

This branch:
```
 1kB  helpers/helpers.d.ts
 2kB  helpers/types.d.ts
22kB  helpers/index.js
122kB  source-map-support/index.js
 0kB  web-handler/web-handler.d.ts
 1kB  web-handler/helpers.d.ts
 2kB  web-handler/types.d.ts
10kB  web-handler/index.js
```

So it's 13kB more